index.html

```diff
@@ -100,6 +110,89 @@ <h2 class="card2_title2">Printing Data</h2>
         <button class="card2button" type="button">See Project</button>
       </div>
 
+      <!-- Card2 Desktop version -->
+      <div class="card2_desktop">
+        <button class="desktop_card2button" type="button">See Project</button>
+      </div>
+
+
+      <!-- Card3 Desktop version -->
+      <div class="card3_card5">
+        <h2 class="card3_card5_title">Data Dashboard</h2>
+        <h2 class="card3_card5_title35">Healthcare</h2>
+        <p class="card3_card5_paragraph">A daily section of privately personalized reads; no accounts or signups
+          required
+          It has been the industry's standard.
+        </p>
+        <ul class="buttons35">
+          <li><button class="li_buttons35" type="button">html</button></li>
+          <li><button class="li_buttons35" type="button">bootstrap</button></li>
+          <li><button class="li_buttons35" type="button">ruby</button></li>
+        </ul>
+      </div>
+
+      <!-- Card6 Desktop version (note that 6 was swapped with 5) -->
+      <div class="card5_second">
```

Comment:
- [ ] Kindly, make sure that you implement the `see project` button for all your project cards in your desktop version. For now, I can see that button just for the first card. Please, implement it for all your cards so that soon by clicking on it, we can see the details about the project.

Reply:
@RolandM99 Hi, I did not add those buttons because in the template 2 given those buttons were not added to the various project apart from the first.
<img width="794" alt="Screenshot 2022-08-25 at 9 18 28 AM" src="https://user-images.githubusercontent.com/106558709/186626759-00e6cf19-217f-4760-adb9-2bbf38c4d9ce.png">

Reply:
i have added them when hovered. thanks